src/test/ui/closure_context/issue-26046-fn-mut.stderr

```diff
@@ -6,6 +6,9 @@ error[E0525]: expected a closure that implements the `Fn` trait, but this closur
 15 | |         num += 1;
 16 | |     };
    | |_____^
+17 | 
```

Comment:
@nikomatsakis that space is to pad to compensate for the multiline markers. We can add a check for empty lines and avoid it.